ahmed-yara-sara/src/components/Dropdown.js

```diff
@@ -0,0 +1,44 @@
+ import React, { Component } from "react";
+ import "react-bootstrap";
+ import {Dropdown } from "react-bootstrap";
+ import { useState,useEffect } from "react";
+
+ export default function DropdownMenu(props) {
+const [Genres,setGenres]=useState([]);
+
+  useEffect(() =>{
+    const TMDB_BASE_URL = "https://api.themoviedb.org/3";
+    const constructUrl = (path, query) => {
+      return `${TMDB_BASE_URL}/${path}?api_key=${atob(
+        "ZDJmYTdhZDFlMjZhZjA4NDdkMzQ5ZDdkYmQ1ZjkzZTU="
+      )}&query=${query}`;
+    };
+    const URL = constructUrl("genre/movie/list","");
+    fetch(URL)
+      .then((genre) => genre.json())
+      .then((data) => {
+      setGenres(data.genres);
+      console.log(data.genres)
+});
+},[URL]);
```

Comment:
do you know what this variable mean in the square Brackets? and you put `URL` there?